.travis.yml

```diff
@@ -37,7 +37,6 @@ before_install:
   - if [ "$TRAVIS_OS_NAME" = "linux" ]; then docker pull lukealonso/raiblocks-ci:latest; fi
 
 script:
-  - if [ -n "$ONE_TIME_TESTS" ]; then ci/check-commit-format.sh; fi
-  - if [ -n "$ONE_TIME_TESTS" ]; then doxygen doxygen.config; fi # TODO also deploy the built HTML
+  - if [ -n "$ONE_TIME_TESTS" ]; then ci/one_time_tests.sh; fi
```

Comment:
We don't want this to kill the whole build, I think.

Reply:
I'd like load test failing to result in a failed build, as we won't recognize it otherwise.